.github/workflows/revert-web-wiki-updates.yml

```diff
@@ -24,7 +24,10 @@ jobs:
         run: git fetch source
       - name: Check for differences
         id: check-diff
-        run: echo "diff=$(git log source/develop..origin/master)" >> $GITHUB_OUTPUT
+        run: 
+          diff=$(git log source/develop..origin/master)
+          echo "{diff/ /\ }"
+          echo "diff=$diff" >> $GITHUB_OUTPUT
```

Comment:
How does this escape the space? The first `echo` doesn't save its output anywhere, and the second `echo` looks to be essentially the same thing as we had before, except we save the output of `$(git log source/develop..origin/master)` into the variable `diff` first. I'm confused about why this works

Reply:
![image](https://github.com/oppia/oppia/assets/85102180/2094ed58-e11f-44b0-adee-ad395f645fd6)
@U8NWXD I assign value to the diff that test still passed. I'm wondering why current test can pass too.

Reply:
@U8NWXD When we directly write echo "$(git log source/develop..origin/master)" >> $GITHUB_OUTPUT
colon will not be treated as string, but when I save result to diff first, then assign to github output, colon become a string now and we don't need to escape it's meaning.
diff=$(git log source/develop..origin/master)
          echo "$diff=diff" >> $GITHUB_OUTPUT